packages/instant/src/components/zero_ex_instant_provider.tsx

```diff
@@ -120,6 +121,15 @@ export class ZeroExInstantProvider extends React.Component<ZeroExInstantProvider
         gasPriceEstimator.getGasInfoAsync();
         // tslint:disable-next-line:no-floating-promises
         this._flashErrorIfWrongNetwork();
+
+        // Analytics
+        analytics.addEventProperties({
+            embeddedHost: window.location.host,
+            embeddedUrl: window.location.href,
+            networkId: state.network,
+            providerName: state.providerState.name,
```

Comment:
as discussed offline, we need some kind of versioning here or else we will not be able to tell if the absence of an event is attributable to an old version or user inaction

Reply:
This is done now